src/analysis/normed_space/weak_dual.lean

```diff
@@ -10,31 +10,36 @@ import analysis.normed_space.operator_norm
 /-!
 # Weak dual of normed space
 
-This file proves properties of the weak-* topology on the dual of a normed space.
+This file proves properties of the weak-* topology on the dual of a normed space `E`.
```

Comment:
```suggestion
This file proves properties of the weak-* topology on the dual ``weak_dual 𝕜 E` of a normed space `E`.
```

Reply:
I don't actually think this is clearer ... "the dual" is both `weak_dual 𝕜 E` and `normed_space.dual 𝕜 E` (they are type synonyms), so not specifying one of the two is deliberate.

Reply:
Exactly as Heather said, I think mathematicians think it is the same dual equipped with two different topologies, and the main result so far is the comparison of the two topologies.

Eric's suggestion would have the advantage of having a link in the docs to the definition, though (but then one should probably also link to `normed space.dual`, since this is the other half of the comparison).

I am totally happy with either. If I had to pick, I'd probably keep the original, because I believe it is the most helpful to mathematicians glancing the contents. But there is a tradeoff (links are also helpful, and that's indeed what I was fixing elsewhere).

Reply:
[Update:] Actually I think we can have the best of both. I'll add the links afterwards once I get back to my usual computer!

Reply:
My point was really just that it's weird to mention `E` without mentioning something else that's a function of `E`. Obviously this file isn't just about spaces called `E`, but when I read this I was expecting a second occurrence of E that never came.

Reply:
Ok! Funnily enough, I actually added `E` there as a supposed fix to a similar issue in the next paragraph after this introductory sentence: the continuation started talking about `weak_dual 𝕜 E` etc. but it had never been mentioned what `E` is (ok, that _could_ be inferred from the arguments of `weak_dual`, but it seems too much to leave to the reader).

I edited, attempting "best of both worlds". This added a bit of length. And the clearest way I came up with was the unimaginative "Let `E` be a normed space over a field `𝕜`".